shelterAPI.json

```diff
@@ -103,6 +100,38 @@
           }
         }
       }
+    },
+    "/shelter/announcements": {
```

Comment:
This change looks a bit odd architecturally, as it appears as though the shelter module has access to announcements. It would be better to use /announcements/shelter instead

Reply:
IMO it's not too bad as it is now, although if we were to change it I think `/announcements/shelter` is kind of unclear. Maybe something like `/announcements/current-shelter` or even `from-current-shelter`?

Reply:
Although honestly I think it would be ok to leave it as it is now. Even if it looks a bit weird for whoever will implement this (since it is a very similar functionality to `/announcements` but has a totally different route), functionally it makes a lot of sense: we have some shelter, and this shelter 'contains' some announcements.

Reply:
Maybe it would be easier to implement an additional optional query parameter for /announcements, like a shelterId=xxx? Would work like filtering and be a little more versatile than creating another endpoint that works specifically with auth claims, for example if a user wanted to see announcements from a specific shelter. Though it would seem a bit weird side to side with the 'locations' filter

Reply:
It has a totally different purpose, so IMO it should be a separate endpoint. Also it's much easier to not bother with passing the ID and just relying on the one stored in the token.

Reply:
> This change looks a bit odd architecturally, as it appears as though the shelter module has access to announcements. It would be better to use /announcements/shelter instead

I agree that shelter module should be separate from announcements module, however it can be achieved architecturally via related services or generally understood business logic. I believe the main aim of endpoint names is that they should be easy to understand what they exactly do. That is why I think `/shelter/announcements` is far more explicit than `\announcements\shelter`. 

As the specification says in the Use case diagram section:
![image](https://user-images.githubusercontent.com/44144898/232295942-473b7c2b-efab-45bf-b991-1e2b8d49218c.png)
it is naturally understood as a functionality tightly coupled with shelter.

Reply:
I agree 🔥✍️🔥

Reply:
But in out microservice architecture it cannot be in shelterAPI because shelterAPI is not allowed to access announcements, so maybe try to move it to announcementsAPI

Reply:
I understand. Please make this change in specification by yourself as it is forced and approved only by your team. Then please merge this PR